agent.go

```diff
@@ -568,12 +568,14 @@ func (a *Agent) updateConnectionState(newState ConnectionState) {
 }
 
 func (a *Agent) setSelectedPair(p *CandidatePair) {
-	a.log.Tracef("Set selected candidate pair: %s", p)
-
 	if p == nil {
+		a.log.Tracef("Reset selected candidate pair")
```

Comment:
`Unset selected candidate pair` or `Set candidate pair to nil` maybe? I would assume that reset was performed upon the candidate pair. Like reseting their state/clearing the count of packets set etc...